docs/ipdk-dpdk.md

```diff
@@ -0,0 +1,123 @@
+# Building IPDK networking recipe for DPDK target
+
+## Overview
+
+This document describes how to build and run IPDK networking recipe on DPDK target.
+
+## Setup
+
+### Build P4 SDE for DPDK
```

Comment:
[MD022 - Headings should be surrounded by blank lines](https://github.com/DavidAnson/markdownlint/blob/v0.26.2/doc/Rules.md#md022)